src/effects/VST/VSTEffect.h

```diff
@@ -73,6 +73,17 @@ typedef SInt16 CFBundleRefNum;
 #endif
 #endif
 
+struct VSTEffectSettings
+{
+   std::unordered_map<wxString, double> mParamsMap;
```

Comment:
Remember `#include <unordered_map>` in the commit that introduces this line, so the macOS build isn't broken.

Reply:
ok, done